### ISSUE TITLE
How the performance of baseline methods are obtained?

### ISSUE THREAD

**Issue by Lafite-Yu** [szc19990412/TransMIL#8]
As the dataset used in this paper is different from the compared baseline methods, how their performance results are obtained? Did you train and eval the baseline models respectively? Thanks.

**Comment by WangZhiKang77** [comment]
> As the dataset used in this paper is different from the compared baseline methods, how their performance results are obtained? Did you train and eval the baseline models respectively? Thanks.

I have the same question. I email him and he didn't reply.

**Comment by Lafite-Yu** [comment]
> > As the dataset used in this paper is different from the compared baseline methods, how their performance results are obtained? Did you train and eval the baseline models respectively? Thanks.
> 
> I have the same question. I email him and he didn't reply.

The performance of other baseline methods reported in the paper is lower than metrics in the original papers, I wonder how the authors acquired such results. However, the reported performance of TransMIL is still higher than the performance of the baseline methods in their own papers.

**Comment by marvinyan080** [comment]
> > As the dataset used in this paper is different from the compared baseline methods, how their performance results are obtained? Did you train and eval the baseline models respectively? Thanks.
> 
> I have the same question. I email him and he didn't reply.

Hi, did you get response from them? I cannot reproduce their results, but for the methods they compared I can reproduce and those methods have higher performance then what they reported in the paper.

**Comment by Lafite-Yu** [comment]
> For the comparison methods, except MIL-RNN, all methods use 1024-dimensional features extracted by ImageNet pre-trained ResNet50, and all comparison schemes use the same training framework pytorch lightning and the same training parameters, such as learning rate, etc. The parameters reported in the comparative method paper depend on the specific parameters used in the experiment, and unfortunately under the same experimental conditions as ours, better results were not achieved.

I think this should not be a fair comparison for other models, may I understand in the following way: you carefully find the best hyper-params setting for your model, while omitting the best setting for other models when reproducing their results?
